doc/protocol.md

```diff
@@ -60,15 +60,15 @@ Both request and response use the same set of messages.
 
 ### Transfer resume
 The GA4GH streaming protocol supports resume of a stream by sending periodic
-checkpoint message that can be used to restart a transfer at that point. See
-the Rational section for discussion of this design decision.  To resume from a
+checkpoint messages that can be used to restart a transfer at that point. See
+the Rationale section for discussion of this design decision.  To resume from a
 failure, a `resume` message, along with a checkpoint object is sent to the
 server. 
 
 The frequency of checkpoint messages can be requested by the client using the
-`X-GA4GH-CHECKPOINT-FREQUENCY` request header.  This is an advisory request,
-The exact frequency of checkpoints is up to the discretion of the server.  The
-value is the number of approximate number of bytes of to stream before sending
+`X-GA4GH-CHECKPOINT-FREQUENCY` request header.  This is an advisory request;
+the exact frequency of checkpoints is up to the discretion of the server.  The
+value is the approximate number of bytes to stream before sending
 a checkpoint message.  The value is an integer with an optional
 case-insensitive suffix of K for kilobytes, M for megabytes, and G for
 gigabytes.  Specifying 0 disables checkpoints.
```

Comment:
Question: do we really need K,M and G suffixes as part of the protocol? Can't the client just multiply these out themselves?